app/templates/pains_verify_and_coefficient_use.html

```diff
@@ -56,14 +56,48 @@ <h3>Verify Pains Filter</h3>
   <br>
     <div class="text-left">
       <form action="\final_compounds" method=POST enctype=multipart/form-data>
-        <script id='include_mpo_script' include_mpo={{include_mpo}}>
-          include_mpo = document.getElementById('include_mpo_script').getAttribute('include_mpo');
-          if (include_mpo == "True") {
-            document.write('Mpo Score 0 Color: <input type="color" name="mpo0Color"/><br>Mpo Score 6 Color: <input type="color" name="mpo6Color"/><br>');
-          } else {
```

Comment:
We still need this code for the minute while we use mpo score as a default option. Last year we allowed the input to be optional.